pontoon/base/tests/test_views.py

```diff
@@ -140,3 +146,42 @@ def test_no_subpage_no_stats_in_current_locale(self):
         with patch('pontoon.base.views.render', wraps=render) as mock_render:
             self.client.get(url)
             assert_true('part' not in mock_render.call_args[0][2])
+
+
+class ContributorsTests(TestCase):
+    @patch('pontoon.base.models.UserTranslationsManager.with_translation_counts')
+    @patch('pontoon.base.views.render', wraps=render)
+    def test_default_period(self, mock_render, mock_translations_manager):
+        """
+        Checks how view handles default period.
```

Comment:
Docstrings on tests are useful for elaborating what should be happening. This one doesn't really explain what you want to happen.

Also, if the docstring is sufficiently short (I use 72 characters myself), you can keep it all on one line to save space.